packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts

```diff
@@ -625,37 +625,50 @@ export class NotifiFrontendClient {
       signMessage,
     } as SignMessageParams;
 
-    try {
-      if (this._userState && this._userState.status === 'authenticated') {
-        await this.logIn(signMessageParams);
-      }
-      const timestamp = Math.round(Date.now() / 1000);
-      const signature = await this._signMessage({
-        signMessageParams,
-        timestamp,
-      });
-      const connectedWallet = await this._service.connectWallet({
-        walletBlockchain,
-        walletPublicKey,
-        accountId:
-          walletBlockchain === 'APTOS' ||
-          walletBlockchain === 'ACALA' ||
-          walletBlockchain === 'NEAR' ||
-          walletBlockchain === 'SUI'
-            ? params.walletParams.accountAddress
-            : undefined,
-        signature,
-        timestamp,
-        connectWalletConflictResolutionTechnique:
-          params.connectWalletConflictResolutionTechnique,
-      });
-      return connectedWallet;
-    } catch (e: unknown) {
```

Comment:
I removed the try-catch in subscribeWallet because errors should be handled in next layer (card)